Fixed Broken links #3335

PrathamAditya opened the pull request.

NotMyFault reviewed (changes requested):
Hey,

thanks for your PR. For future reference, please create a branch on your fork of this repository, where you commit your changes to, rather than using the default branch.

Thanks.

MarkEWaite reviewed (approved):
@kmartens27 and @NotMyFault I removed the changes that were breaking links and updated the PostgreSQL link to note that the plugin is unreleased.

Be sure that this is squash merged.

MarkEWaite commented:
@PrathamAditya you can also make it easier on others if you create the pull request from a branch that is named something other than `master`.  The `master` branch on your local repository is best kept as matching the upstream master branch.

MarkEWaite commented on content/blog/2018/07/2018-07-17-simple-pull-request-plugin.adoc:
```diff
@@ -193,13 +193,13 @@ Video:
 * link:https://github.com/jenkinsci/simple-pull-request-job-plugin[Project repository]
 * link:/projects/gsoc/2018/simple-pull-request-job-plugin/[Project page]
-* link:https://app.gitter.im/#/room/#jenkinsci_simple-pull-request-job-plugin:gitter.im[Gitter chat]
+* link:https://app.gitter.im/#/room/#jenkinsci_simple-pull-request-job-plugin[Gitter chat]
```
Resolved by reverting the change

PrathamAditya replied:
> No, we won't merge a PR which breaks URLs just to file another PR to revert that. Please drop these changes proposed. The links are valid as they are presently.

I am sorry for inconvenience and I will make sure it won't happen again

PrathamAditya replied:
> Resolved by reverting the change

Thank you

PrathamAditya commented:
> @kmartens27 and @NotMyFault I removed the changes that were breaking links and updated the PostgreSQL link to note that the plugin is unreleased.
> 
> Be sure that this is squash merged.

Thank you so much
